include/bluetooth/audio.h

```diff
@@ -921,7 +921,7 @@ int bt_audio_chan_stop(struct bt_audio_chan *chan);
  *  source channel.
  *
  *  Broadcast sink channels shall be released using
```

Comment:
It sound weird to say that they shall be released using a delete function.
Propose to use same wording as for source (just below): "cannot be released, but can be deleted [...]"

Reply:
A missing "d" again ... 
(Nitpicking, I know.  But it looks better without spelling errors.)

Reply:
Odd that I keep making that typo... Oh well, it's fixed :D